app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -152,36 +187,29 @@ class Identification extends React.Component {
           for (var issue of err.inner) {
             issues[issue['path']] = issue['errors'];
           }
-          self.setState({ errors: issues });
+          this.setState({ errors: issues });
         }
       });
   };
 
-  getLanguageOptions() {
-    const langOptions = supportedLanguages.languages.map(lang => {
-      const fullySupported = lang.supported.sms && lang.supported.email && lang.supported.phone;
-      const langLabel = fullySupported ? lang.name : lang.name + '*';
-      return { value: lang.name, label: langLabel };
-    });
-    return langOptions;
-  }
-
-  getLanguageValue = language => {
-    return this.state.languageOptions.find(lang => lang.value === language);
-  };
-
-  renderPrimaryLanguageSupportMessage(selectedLanguage) {
-    if (selectedLanguage) {
-      const languageJson = supportedLanguages.languages.find(l => l.name === selectedLanguage);
+  /**
+   * @param {Object} language - in the format of { c: isoCode, d: displayName }
```

Comment:
just a comment, can we call these `code` and `display` or something so you know what they are right off the bat

Reply:
sure! I like that change